src/goto-instrument/alignment_checks.cpp

```diff
@@ -19,21 +19,21 @@ void print_struct_alignment_problems(
   const symbol_tablet &symbol_table,
   std::ostream &out)
 {
-  forall_symbols(it, symbol_table.symbols)
-    if(it->second.is_type && it->second.type.id()==ID_struct)
+  for(const auto &symbol_pair : symbol_table.symbols)
+  {
+    if(symbol_pair.second.is_type && symbol_pair.second.type.id() == ID_struct)
     {
-      const struct_typet &str=to_struct_type(it->second.type);
-      const struct_typet::componentst &components=str.components();
+      const struct_typet &str = to_struct_type(symbol_pair.second.type);
+      const struct_typet::componentst &components = str.components();
 
-      bool first_time_seen_in_struct=true;
+      bool first_time_seen_in_struct = true;
 
-      for(struct_typet::componentst::const_iterator
-          it_mem=components.begin();
-          it_mem!=components.end();
+      for(struct_typet::componentst::const_iterator it_mem = components.begin();
+          it_mem != components.end();
           it_mem++)
```

Comment:
Why all these whitespace changes? Makes it harder to see what are actual code changes. If cleaning up the formatting is a good idea, can you do this in a separate commit?

Reply:
Because I ran my updates through clang-format before making each commit. I now realise these should have been kept in separate commits.

Reply:
@hannes-steffenhagen-diffblue If you add "?w=1" to the end of the URL when viewing a diff on github then it ignores whitespace when making the diff.

Reply:
Add to that a later use of `git blame` also requires an additional `-w`. So, yes, there are workarounds, but maybe the pain can be avoided in the first place?

Reply:
I agree that running clang-format should be done in a separate commit in this instance, as it will make the PR much easier to review. Ignoring whitespace when viewing a diff is still useful in other situations, e.g. when a conditional is introduced which changes the indentation of a large section of code.

Reply:
Though with that flag you can no longer add comments seemingly